_studio/mfx_lib/fei/h264_la/mfx_h264_la.cpp

```diff
@@ -614,8 +614,7 @@ mfxStatus VideoENC_LA::InsertTaskWithReordenig(sLAInputTask &newTask, bool bEndO
     { 
         m_numLastB = 0;
         m_miniGop.resize(0);
-        sLAInputTask finished_task = {0};
-        m_syncTasks.push_back(finished_task);    
+        m_syncTasks.emplace_back();
```

Comment:
sLAInputTask doesn't have user defined constructor, so it won't be initialized, but code before assumed zero-initialized object. Pass empty initializer list as constructor argument, it should make the trick:
```
m_syncTasks.emplace_back(static_cast<std::initializer_list<sLAInputTask> >({})); // maybe you'll need slightly other cast, but I think this one should work
```